app/src/components/widgets/singleLevelWidgets/tables/flakyTests/flakyTests.jsx

```diff
@@ -19,8 +19,15 @@ import PropTypes from 'prop-types';
 import { TestsTableWidget } from '../components/testsTableWidget';
 import * as cfg from './flakyTestsCfg';
 
+const prepareWidgetData = ({ flaky }) =>
+  flaky.map((item) => ({ ...item, statuses: item.statuses.reverse() }));
```

Comment:
`reverse` mutates the array, so please create a new one to prevent bugs:
`[...item.statuses].reverse()`